frontend/content/posts/creating-lambda-layers-with-cdk-and-typescript/index.yml

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
I believe this doc separator is causing build failures. Can you remove?

Reply:
Absolutely, removed.

Reply:
@mbonig Figured it out, I needed an author folder for it to reference.